package.json

```diff
@@ -7,13 +7,16 @@
     "deploy-staging": "sh scripts/deploy-staging.sh",
     "deploy-storybook": "storybook-to-ghpages",
     "deploy": "sh scripts/deploy.sh",
+    "generate-types": "apollo codegen:generate --target typescript --outputFlat src/v2/apollo/types",
+    "generate-types:watch": "yarn generate-types --watch",
     "jest": "jest",
     "lint": "eslint --ext .js,.jsx,.ts,.tsx ./src --fix",
     "mocha": "sh scripts/mocha.sh",
     "predeploy": "sh scripts/predeploy.sh",
     "restart-dyno": "node bin/restart",
     "start-dev": "node src/index.js",
     "storybook": "start-storybook -p 9001 -c .storybook",
+    "sync-schema": "yarn run apollo schema:download src/v2/apollo/schema.json",
```

Comment:
Saw that y'all have a `schema.json` and `schema.graphql` in the `v2/apollo` folder. Figured this would be a handy script to have. Hooks into `apollo.config.js` in the project root, and requires that a `GRAPHQL_XAPP_TOKEN` be set in `.env`.